src/hooks/imagePicker.ts

```diff
@@ -45,3 +47,40 @@ export const useSelectImage = (
 
   return { imageUri, selectImage };
 };
+
+export const useCaptureImage = (
+  onChange?: <T>(
+    setter: React.Dispatch<React.SetStateAction<T>>
+  ) => React.Dispatch<React.SetStateAction<T>>,
+  allowsEditing?: boolean,
+  aspect?: [number, number],
+  quality?: number,
+  mediaTypes?: MediaTypeOptions
+) => {
+  const [imageUri, setImageUri] = useState<string>();
+
+  const captureImage = useCallback(async () => {
+    const { status } = await requestCameraPermissionsAsync();
+
+    if (status !== PermissionStatus.GRANTED) {
+      Alert.alert(texts.errors.image.title, texts.errors.image.body);
+      return;
+    }
+
+    // this allows for proper selecting and cropping to 1:1 images (and not videos)
+    // for more details about options see: https://docs.expo.dev/versions/latest/sdk/imagepicker/#imagepickermediatypeoptions
+    const result = await launchCameraAsync({
+      mediaTypes: mediaTypes ?? MediaTypeOptions.Images,
+      allowsEditing: allowsEditing ?? true,
```

Comment:
i suggest fallback `false` here

Reply:
Since it is `true` in the `useSelectImage` function, I set it to `true` here.

<img width="327" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/9a1c27e3-867e-4870-b658-736512216f52">

Reply:
This screenshot is the section in `useSelectImage` where we set `allowsEditing`. Here I wanted to show that we set it to true before, but now I set it to false by default because we are not using the image editing feature.

https://github.com/smart-village-solutions/smart-village-app-app/pull/1007/commits/f9c2849385f73cb7ac9388da8f85a0675817a9aa